unique_toolkit/unique_toolkit/language_model/service.py

```diff
@@ -73,6 +73,25 @@ def from_magic_table_event(cls, event: MagicTableEvent) -> "LanguageModelService
             user_id=event.user_id,
         )
 
+    @classmethod
```

Comment:
Umm... seems that the init method does exaclty the same...

Reply:
agree